.gitignore

```diff
@@ -284,3 +284,4 @@ gradle-app.setting
 # End of https://www.gitignore.io/api/java,linux,macos,gradle,windows,eclipse,intellij+all,visualstudiocode
 
 /sam-app/
+datacite-mds-test-config.json
```

Comment:
This file is shared in lastpass, currently holds one valid client configuration. I use it together with the CLI tool.